assets/src/edit-story/components/inspector/prepublish/checklistTab.js

```diff
@@ -0,0 +1,199 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * WordPress dependencies
+ */
+import { __, sprintf } from '@wordpress/i18n';
+
+/**
+ * External dependencies
+ */
+import { useCallback, useMemo, Fragment } from 'react';
+import styled from 'styled-components';
+import { v4 as uuidv4 } from 'uuid';
+
+/**
+ * Internal dependencies
+ */
+import { SimplePanel } from '../../panels/panel';
+import { Rectangle } from '../../../icons';
+import { PRE_PUBLISH_MESSAGE_TYPES } from '../../../app/prepublish/constants';
+import { GuidanceChecklist } from '../../../app/prepublish/types';
+import { Label } from '../../form';
+
+const NumberDecorator = styled.span`
+  display: inline-flex;
+  justify-content: center;
+  align-items: center;
+  height: 20px;
+  width: 20px;
+  font-size: ${({ theme }) => theme.fonts.body1.size};
+  margin-right: 7px;
+  border-radius: 100%;
+  color: ${({ theme }) => theme.colors.bg.panel};
+  background-color: ${({ theme, error }) =>
+    error ? theme.colors.fg.negative : theme.colors.fg.warning};
+`;
+
+const PanelTitle = styled.span`
+  color: ${({ theme, error }) =>
+    error ? theme.colors.fg.negative : theme.colors.fg.warning};
+`;
+
+const Row = styled.div`
```

Comment:
Yeah, trying to figure out the nicest way for this to look for now... Since I'm not going to change the styles for the panels (i.e. the chevron is on the wrong side), the alignment will be different from the Figma. It could definitely look nicer though so I'll change it.

Let's see what it looks like when the rectangle aligns with the headings of each row... otherwise it throws off the alignment between the tab panels e.g.:  
![Screen Recording 2020-11-17 at 5 08 28 PM](https://user-images.githubusercontent.com/41136059/99465526-dd506a00-28f7-11eb-8d5b-39b0333ec25b.gif)

Reply:
It's kind of awkward with there being some items underneath the page headers and some not. I think the High Priority/Recommended headers should stay left, then align the rest with the sub-headers. In figma it doesn't show what it looks like to have story-level guidelines. Might it make sense to nest the story-level ones underneath a story header?

Reply:
This does look a lot better though and seems good enough for this release. I'm just coming up with design ideas now so feel free to ignore me. Having them aligned like this is much better than before.